docs/organizations/using-a-coding-standard.md

```diff
@@ -39,9 +39,8 @@ To create a coding standard for your organization:
 
     !!! tip
         -   You can use the filters to help you find the relevant tools and code patterns. The recommended configurations are manually curated by Codacy or based on the default code patterns of the tools and are marked with the icon ![Recommended icon](images/coding-standard-recommended-icon.png).
-        -   To toggle multiple code patterns at once, click the checkbox of the first pattern and **Shift+click** the checkbox of the last pattern in a range.
-
-            Alternatively, click the checkbox on the header of the code pattern list to toggle all code patterns that are currently visible on the list.
+        -   To toggle multiple code patterns at once, you can click the checkbox of the first pattern and **Shift+click** the checkbox of the last pattern in a range.
```

Comment:
I think I know why you added the "you can" here (to make it more parallel with the next item), but I don't think it's really necessary since the next item also starts with "To toggle all the code patterns visible on the list, click the checkbox [...]". Avoiding the "you can" makes for a more assertive and terser writing style overall.

```suggestion
        -   To toggle multiple code patterns at once, click the checkbox of the first pattern and **Shift+click** the checkbox of the last pattern in a range.
```